src/test/scala/scalismo/faces/color/RGBTests.scala

```diff
@@ -16,40 +16,18 @@
 
 package scalismo.faces.color
 
-import java.awt.Color
-
 import scalismo.faces.FacesTestSuite
 
 class RGBTests extends FacesTestSuite {
 
   describe("RGB") {
-    it("can be converted to Java AWT Color") {
-      val awtColor = Color.CYAN
-      RGB(0, 1, 1).toAWTColor shouldBe awtColor
-    }
-
-    it("can be created from a Java AWT Color") {
-      val awtColor = Color.CYAN
-      RGB(awtColor) shouldBe RGB(0, 1, 1)
-    }
-
     it("supports a roundtrip conversion to/from AWT Color") {
       val color = randomRGB
       RGB(color.toAWTColor).toAWTColor shouldBe color.toAWTColor
     }
   }
 
   describe("RGBA") {
-    it("can be converted to Java AWT Color") {
-      val awtColor = Color.CYAN
-      RGBA(0, 1, 1).toAWTColor shouldBe awtColor
-    }
-
-    it("can be created from a Java AWT Color") {
-      val awtColor = Color.CYAN
-      RGBA(awtColor) shouldBe RGBA(0, 1, 1)
-    }
-
     it("supports a roundtrip conversion to/from AWT Color") {
       val color = randomRGBA
       RGBA(color.toAWTColor).toAWTColor shouldBe color.toAWTColor
```

Comment:
As above, this tests more than a roundtrip. This might lead to that some errors are not detected.